README.md

```diff
@@ -107,8 +107,8 @@ best_it_commercetools_filter:
             # Prototype
             key:
 
-                # Api query for sdk
-                query:                ~ # Required
+                # Api query for sdk (default: null for relevance sorting) 
```

Comment:
Das würde ich mit in die Konfigurationsklasse schreiben. Damit kann man sich den Konfigurationsdump dann generieren.

Reply:
Guter Punkt. Ergänze ich morgen